linter.py

```diff
@@ -10,40 +10,20 @@
 
 """This module exports the HamlLint plugin class."""
 
-import os
-from SublimeLinter.lint import RubyLinter, util
+from SublimeLinter.lint import RubyLinter
 
 
 class HamlLint(RubyLinter):
     """Provides an interface to haml-lint."""
 
     syntax = 'ruby haml'
     cmd = 'ruby -S haml-lint'
-    version_args = '-S haml-lint --version'
-    version_re = r'(?P<version>\d+\.\d+\.\d+)'
-    version_requirement = '>= 0.6.0'
     regex = r'^.+?:(?P<line>\d+) \[(:?(?P<warning>W)|(?P<error>E))\] (?P<message>.+)'
     tempfile_suffix = 'haml'
-    config_file = ('--config', '.haml-lint.yml')
+    
+    defaults: {
+        'env': {
+            'HAML_LINT_RUBOCOP_CONF': '${folder}/.rubocop.yml'
+        }
+    }
 
-    def build_args(self, settings):
```

Comment:
You MUST not override build_args. The code is also not thread-aware. We only guard the instantiation of the class (basically __init__).